src/app/components/SearchResults/ResultsList.jsx

```diff
@@ -83,17 +86,36 @@ class ResultsList extends React.Component {
       const allEditionsLink = getEditionsLinkElement(result);
       const previewEdition = result.editions && result.editions[0];
 
+      const citationData = CitationFormatter.getCitationData(result, result.editions ? result.editions[0] : {});
+      const citationElement = shouldShowCitations ? (
+        <APACitation
+          title={citationData.title}
+          subTitle={citationData.sub_title}
+          agents={citationData.agents}
+          publicationYear={citationData.publication_year}
+          edition={citationData.edition_statement}
+          volume={citationData.volume}
+          sourceLink={citationData.sourceLink.link}
+          isGovernmentDoc={citationData.isGovermentDoc}
+        />
+      ) : undefined;
+
+
       return (
-        <DS.SearchResultItem
-          id={`search-result-${result.uuid}`}
+        <div
           key={`search-result-${result.uuid}`}
-          resultIndex={index}
-          headingContent={titleElement}
-          subtitleContent={EditionCard.getSubtitle(result.sub_title)}
-          authorLinkElement={authorLinkElement ? joinArrayOfElements(authorLinkElement, ', ') : undefined}
-          editionInfo={EditionCard.getEditionData(result, previewEdition, eReaderUrl, referrer, showRequestButton)}
-          editionsLinkElement={allEditionsLink}
-        />
+        >
+          <DS.SearchResultItem
+            id={`search-result-${result.uuid}`}
+            resultIndex={index}
+            headingContent={titleElement}
+            subtitleContent={EditionCard.getSubtitle(result.sub_title)}
+            authorLinkElement={authorLinkElement ? joinArrayOfElements(authorLinkElement, ', ') : undefined}
+            editionInfo={EditionCard.getEditionData(result, previewEdition, eReaderUrl, referrer, showRequestButton)}
+            editionsLinkElement={allEditionsLink}
+          />
+          {citationElement}
```

Comment:
Instead of this, you can go `{shouldShowCitations && <APACitation {... all the props } />`